apps/condo/domains/ticket/utils/handlers.spec.js

```diff
@@ -77,4 +81,44 @@ describe('Ticket request event detection', () => {
         expect(detectEventTypes(requestData)).toMatchObject({ [EXECUTOR_CONNECTED_EVENT_TYPE]: true })
     })
 
+    it('correctly detects event on ticket create', () => {
+        const id = faker.datatype.uuid()
+        const requestData = {
+            operation: 'create',
+            existingItem: null,
+            updatedItem: {
+                client: id,
+            },
+        }
+
+        expect(detectEventTypes(requestData)).toMatchObject({ [STATUS_CHANGED_EVENT_TYPE]: false } )
+    })
+
+    it('correctly detects event connection on ticket create', () => {
+        const id = faker.datatype.uuid()
+        const requestData = {
+            operation: 'create',
```

Comment:
please make sure data you're using for tests is emulatiing real data structure and values